File: client-angular/src/app/api/models/audit-dto.ts
/* tslint:disable */
/* eslint-disable */
import { AnswerDto } from '../models/answer-dto';
import { AuditActionDto } from '../models/audit-action-dto';
export interface AuditDto {
  actions: Array<AuditActionDto>;
  answers: Array<AnswerDto>;
  area: string;
  auditId: string;
  author: string;
  endDate: string;
  score: number;
  startDate: string;
}

File: api/Features/Audit/Dto/AuditDto.cs
using System;
using System.Collections.Generic;

namespace Features.Audit.Dto;

public sealed record AuditDto
{
    public required Guid AuditId { get; init; }
    public required string Author { get; init; }
    public required string Area { get; init; }
    public required DateTime StartDate { get; init; }
    public required DateTime EndDate { get; init; }
    public required double Score { get; init; }
    public required IReadOnlyList<Answer.Dto.AnswerDto> Answers { get; init; }
    public required IReadOnlyList<AuditAction.Dto.AuditActionDto> Actions { get; init; }
}
